### PR TITLE
build: Moving files from Alberto and Gwen

### PR DESCRIPTION
**Pull request by stanleyjones**
Populating the `lifted-ui` repository with the `shared/` folder created in https://github.com/liftedinit/gwen/pull/8 and https://github.com/liftedinit/albert/pull/98.

- [x] Copy over files
- [x] Setup package details
- [x] Get tests running locally

### CODE REVIEW

**Comment by hansl** [comment]
Ping me when this is ready for review (this is draft).

File: helpers/__tests__/convert.test.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    for (let j = 0; j < buffer.byteLength; j++) {
+      buffer[j] = Math.random() * 256;
+    }
+
+    let encoded: E = encoder(buffer.buffer as T);
+    let decoded: T = decoder(encoded);
+
+    let decodedBuffer = ArrayBuffer.isView(decoded)
+      ? decoded
+      : Buffer.from(decoded);
```
Comment (stanleyjones) [inline helpers/__tests__/convert.test.ts]: **NOTE**: Actual logical change here.

The test was failing with the following error:
```
    helpers/__tests__/convert.test.ts:17:37 - error TS2769: No overload matches this call.
      The last overload gave the following error.
        Argument of type 'BufferSource' is not assignable to parameter of type 'WithImplicitCoercion<string> | { [Symbol.toPrimitive](hint: "string"): string; }'.
          Type 'ArrayBufferView' is not assignable to type 'WithImplicitCoercion<string> | { [Symbol.toPrimitive](hint: "string"): string; }'.

    17     let decodedBuffer = Buffer.from(decoded);
```

Which makes me wonder if the test was actually running when we added it to Alberto.

**Comment by stanleyjones** [comment]
There's still some work to do here. Alberto and Gwen import the Chakra theme slightly differently so I need to clean that up. My plan is to merge this PR so I have something to work with, then import it from Alberto and Gwen in new branches, then come back and adjust this repo until the two apps converge.

File: helpers/__tests__/convert.test.ts (hunk 1 above)
Comment (hansl) [reply helpers/__tests__/convert.test.ts]: I made sure the test was failing before fixing it. So yes, it was running. I'm wondering if you have the right versions of dependencies.

Comment (stanleyjones) [reply helpers/__tests__/convert.test.ts]: The testing environment now mirrors the Alberto setup and the original tests pass. Sorry for the confusion.

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "arrowParens": "avoid",
+  "bracketSpacing": true,
+  "embeddedLanguageFormatting": "auto",
+  "endOfLine": "lf",
+  "insertPragma": false,
+  "printWidth": 80,
```
Comment (hansl) [inline .prettierrc:7]: I feel like we should revisit some of those, but it's outside the scope of this PR.

**Review by hansl — APPROVED** [review]
LGTM. I assume all of this is moved over directly from gwen/alberto.

File: .prettierrc (hunk 2 above)
Comment (stanleyjones) [reply .prettierrc:7]: Agreed on both points.

**Pull request merged**